resources/js/controllers/listener_controller.js

```diff
@@ -44,7 +44,15 @@ export default class extends ApplicationController {
                 }
             }));
 
-        this.asyncLoadData(params);
+        this.asyncLoadData(params).then(() => {
+            document.dispatchEvent(
+                new CustomEvent("orchid:listener:post-render", {
```

Comment:
What do you think about renaming it to `orchid:listener:after-render for more clarity?

Reply:
sure!